<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1920</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>20</width>
      <height>981</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Operations</string>
    </property>
    <widget class="QPushButton" name="select">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Select Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="process">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Process Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="extract">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Extract Plate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="match">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Match Plate Details</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Clear </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Selected Image</string>
    </property>
    <widget class="QLabel" name="selectedFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string> Processed Image 1(gray)</string>
    </property>
    <widget class="QLabel" name="gray_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>90</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Processed Image 2 (edge)</string>
    </property>
    <widget class="QLabel" name="edge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Processed Image 3(Threshold)</string>
    </property>
    <widget class="QLabel" name="threshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Final Image(Number Plate)</string>
    </property>
    <widget class="QLabel" name="np">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>340</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Extracted Number plate</string>
    </property>
    <widget class="QPlainTextEdit" name="Extract_det">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif Collection</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>600</y>
      <width>1761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>610</y>
      <width>641</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Matched Plate Details</string>
    </property>
    <widget class="QPlainTextEdit" name="matched">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>621</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
